src/main/java/entities/Hobbies.java

```diff
@@ -0,0 +1,13 @@
+package entities;
+
+public enum Hobbies {
```

Comment:
do we need this to be enum?

Reply:
Our matcher matches users who are at same age, and share the same program of study and hobbies and whom meets the request user’s gender orientation. To avoid people entering various types of hobbies that actually mean the same thing (e.g. sports VS sport), we pre-determined the hobbies and have a drop-down menu for users to select on the registration page. Those pre-determined hobbies for selection are reading, travelling, music, games, sports, food, movies and TV, and others. As a result, hobbies in our case are in fact enum categories.